app/assets/javascripts/models/beta/story.js

```diff
@@ -330,27 +330,38 @@ export const mergeWithFetchedStories = (
   fetchedStories,
   pastStoryIds
 ) => {
-  const storyNotSaved = currentStories.filter(isNew);
-
-  const editedStories = fetchedStories.map((fetchedStory) => {
-    const existingStory = currentStories.find(
-      (story) => story.id === fetchedStory.id
-    );
-    if (existingStory && !existingStory.collapsed) {
-      return {
-        ...fetchedStory,
+  const mergedStories = { ...fetchedStories };
+
+  if (Object.values(currentStories).length === 0) {
+    return mergedStories;
+  }
+
+  if (currentStories.stories[null]) {
```

Comment:
We need to find a way to accept multiple "null" stories instead of just one with this change.